### PR TITLE
added model and grid components

### PR DESCRIPTION
**Pull request by anshtripathi01**
**Please review**

### CODE REVIEW

File: Components/Grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.grid-container{
+    box-shadow: 0 0 2px 1px rgba(0,0,0,0.3);
```
Comment (amirnathani9) [inline Components/Grid/grid.css:2]: if the same shadow color is repeating everywhere make a variable of this and use it or comment after it that its used here only

File: Components/Model/model.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
+
+.modal{
+    width: 300px;
+    height: 240px;
+    box-shadow: 0 0 4px 1px rgb(0,0,0,0.3);
+    border-radius: 0.2rem;
```
Comment (amirnathani9) [inline Components/Model/model.css:6]: same goes for here for box shadow

**Review by amirnathani9 — APPROVED** [review]
Some neatpeaks are suggested upper other than that good to merge.
and some suggestions are add discription about what is done in this pr

File: Documentation/docs.css [hunk 3]
```diff
@@ -97,7 +99,7 @@ h3 {
 footer {
     position: absolute;
     display: flex;
-    margin-top: 900vh;
+    margin-top: 970vh;
```
Comment (shashikantmaurya0007) [inline Documentation/docs.css:102]: instead of giving margin to footer instead you can give min-height to your content

**Review by shashikantmaurya0007 — APPROVED** [review]
i have suggested one change rest looks good

**Pull request merged**